### PR TITLE
shark plushie addition

### PR DESCRIPTION
**Pull request by LordBarkBread**
#### Summary
Content "add a shark plushie"

#### Purpose of change
I bring you a large shark plushie, one that our weary and tired survivors may keep on them or on their bedding similar to a teddy bear as a SLEEP_AID item. Wouldn't you like to hug a plush shark for protection while you sleep from the horror that is the cataclysm?

#### Describe the solution
Add a new generic item, shark_plushie
Give the flag SLEEP_AID to plushie item

#### Describe alternatives you've considered
~~Add an even bigger plushie~~

#### Testing
image

#### Additional context
Will need to add spawn locations.

### CODE REVIEW

File: data/json/items/generic.json [hunk 1]
```diff
@@ -2832,6 +2832,20 @@
     "volume": "850 ml",
     "flags": [ "SLEEP_AID" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "shark_plushie",
+    "symbol": "s",
+    "color": "cyan",
+    "name": { "str": "shark plushie" },
+    "description": "A somewhat large, mass produced shark plushie. Meant to provide comfort to people of all ages.",
```
Comment (anothersimulacrum) [inline data/json/items/generic.json:2841]:
```suggestion
    "description": "A somewhat large, mass produced shark plushie.  Meant to provide comfort to people of all ages.",
```

File: data/json/items/generic.json [hunk 2]
```diff
@@ -2832,6 +2832,20 @@
...
+    "id": "shark_plushie",
+    "symbol": "s",
+    "color": "cyan",
+    "name": { "str": "shark plushie" },
+    "description": "A somewhat large, mass produced shark plushie.  Meant to provide comfort to people of all ages.",
+    "price": 600,
+    "price_postapoc": 20,
+    "material": [ "cotton" ],
+    "weight": "640 g ",
+    "volume": "1300 ml",
```
Comment (dseguin) [inline data/json/items/generic.json:2846]:
```suggestion
    "volume": "30000 ml",
```
Not super scientific, but if IKEA says it's 1.03m long, and if you treat it like a cylinder about 0.2m wide, the volume would be about 30 liters

**Comment by Aerin-of-the-Toast** [comment]
blahaj my beloved

**Comment by Daved27hundred** [comment]
You should give them a chance to spawn in furniture stores.

**Comment by LordBarkBread** [comment]
> You should give them a chance to spawn in furniture stores.

It is already set to have a chance to spawn in bed furniture itemgroup, so it should show up in Home Improvement Stores.

**Pull request merged**